src/NUnitFramework/tests/Attributes/TestFixtureSourceTests.cs

```diff
@@ -44,9 +44,12 @@ public class TestFixtureSourceTests
         [TestCase(typeof(SourceReturnsFixtureParameters))]
         [TestCase(typeof(ExtraTestFixtureAttributeIsIgnored))]
         [TestCase(typeof(TestFixtureMayUseMultipleSourceAttributes))]
+        [TestCase(typeof(DerivedClassUsingBaseClassDataSource))]
+        [TestCase(typeof(BaseClassUsingDerivedClassDataSource))]
```

Comment:
👍 Thanks for the thorough test cases!

Reply:
Test Driven Development. First create a test case that shows the problem, then fix it.